frontend/packages/data-layout/src/manager.tsx

```diff
@@ -76,21 +84,24 @@ interface DataManager {
   assign: (key: string, value: object) => void;
   hydrate: (key: string) => void;
   update: (key: string, value: object) => void;
+  reset: () => void;
 }
 
 const defaultTransform = (data: object): object => data;
 const defaultErrorTransform = (err: Error): string => {
   return err?.response?.displayText ?? err.message;
 };
 
-const useDataLayoutManager = (layouts: ManagerLayout): DataManager => {
-  const initialState = {};
+const initialLayoutStepState = { data: {}, isLoading: true, error: null };
```

Comment:
why is isLoading true the default?

Reply:
Ah good point. I think this gets reset in the layout itself as false [here](https://github.com/lyft/clutch/blob/main/frontend/packages/data-layout/src/layout.tsx#L50). We are definitely going to have to come back and clean up the component.